### PR TITLE
Modifying Vault Contracts

### PR DESCRIPTION
**Pull request by wangjj250wo**
Hi, we have made changes to the Vault contract this time.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/loxodrome-perp exports TVL: 
        
 
 
    iotex                     14.80 k
    
    total                    14.80 k

**Comment by llamatester** [comment]
The adapter at projects/loxodrome exports TVL: 
        
 
 
    iotex                     351.28 k
    
    total                    351.28 k

File: projects/loxodrome/index.js
Comment (g1nt0ki) [inline projects/loxodrome/index.js:1]: this leads to doublecount

**Comment by g1nt0ki** [comment]
I have updated the contract now

**Pull request closed without merging**